src/Form/PathautoAdminDelete.php

```diff
@@ -54,9 +54,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     foreach ($objects as $internal_name => $label) {
       $count = db_query("SELECT count(1) FROM {url_alias} WHERE source LIKE :src", array(':src' => "$internal_name%"))->fetchField();
+      // This label is sent through t() in the hard coded function where it is defined.
```

Comment:
This label doesn't make much sense if it is outside of the array. Either move it directly above the assignment, which is OK per coding standards or make it $label or The label.